integration-cli/docker_cli_service_logs_experimental_test.go

```diff
@@ -213,3 +198,51 @@ func (s *DockerSwarmSuite) TestServiceLogsFollow(c *check.C) {
 
 	c.Assert(cmd.Process.Kill(), checker.IsNil)
 }
+
+func (s *DockerSwarmSuite) TestServiceLogsTaskLogs(c *check.C) {
+	testRequires(c, ExperimentalDaemon)
+
+	d := s.AddDaemon(c, true, true)
+
+	name := "TestServicelogsTaskLogs"
+	replicas := 2
+
+	out, err := d.Cmd(
```

Comment:
Doesn't exactly work in this place b/c I'm expecting a service ID, which I can't do a simple string match on. But I understand the gist of what you're saying, I'll fix it.